src/blenderbim/blenderbim/bim/module/pset/ui.py

```diff
@@ -139,6 +140,11 @@ def draw_psetqto_ui(context, pset_id, pset, props, layout, obj_type):
 def draw_psetqto_editable_ui(box, props, prop):
     row = box.row(align=True)
     draw_property(prop, row, copy_operator="bim.copy_property_to_selection")
+    qto_name = ObjectQtosData.data['qtos'][0]['Name']
```

Comment:
The [0] looks suspicious to me. Why are you fetching that index?

Reply:
:-) just because it is a list with all qtos assigned to the element, but an element can only has one pset_qto afaik...so the zero